aecore_eqc/txs_glue_eqc.erl

```diff
@@ -1918,3 +1910,17 @@ fake_contract_id() ->
                    abi = nat(),
                    vm = nat()
                   }).
+
+setup_data_dir() ->
+    %% make sure we can run in eqc/aecore_eqc
+    {ok, Dir} = file:get_cwd(),
+    undefined = application:get_env(setup, data_dir),
```

Comment:
The reason not to do this is that when we work in the Erlang shell (experimenting), then the property may fail misserably and teardown is skipped. Next start we get an error, although we are not redefining data dir.

Since there is a way to undefine application env in the teardown (as you use), I prefer that that is used without checking on beforehand that it is undefined in this hard way.

Reply:
Addressed in cac383b5